### PR TITLE
{phys}[intel/2019b] Elk v6.3.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/285a09b183506911f12e3869e0a3f6a4 for a full test report.

File: easybuild/easyconfigs/e/Elk/Elk-6.3.2-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+dependencies = [
+    ('libxc', '4.3.4'),
+]
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
+prebuildopts += 'echo "LIB_libxc = $EBROOTLIBXC/lib/libxcf90.a $EBROOTLIBXC/lib/libxc.a" >> make.inc && '
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/e/Elk/Elk-6.3.2-intel-2019b.eb:35]: No it ends up in lib on debian/ubuntu systems.

I intended to try using just -lxcf90 -lxc but I forgot. I'll retry with that and update if it works.

[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/e/Elk/Elk-6.3.2-intel-2019b.eb]: And it worked, so updated.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/1006d1ebefaa464b3cbf8ce9af21a52f for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node119.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/4612af924751bea1490b7519c1041116 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @akesandgren !

**Pull request merged**